src/config.rs

```diff
@@ -347,6 +352,64 @@ macro_rules! create_config {
                 }
             }
 
+            /// Construct a `Config` from the toml file specified at `file_path`.
+            ///
+            /// This method only looks at the provided path, for a method that
+            /// searches parents for an `rls.toml` see `resolve_config`.
```

Comment:
Is this line a typo, or only relevant when using from rls? If so, I would like to have it explained here since it may confusing afterward.